### DISCUSSION TITLE
Unable to install openssh in Linux server

### DISCUSSION THREAD

**Discussion by Nitish-Bhatt** [openssl/openssl#22515] [Q&A]
I am unable to proceed further:

Install/upgrade OpenSSL
Issue these commands to install or upgrade the OpenSSL.

$ sudo make
$ sudo make test
$ sudo mv /usr/bin/openssl ~/tmp (Backup current openssl binary)
$ sudo make install

**Comment by paulidale** [comment]
Installing OpenSSL over the top of your system version is very dangerous.  I'd suggest reinstalling the system one.

Note: it's more than just the OpenSSL binary, there are libraries that get installed as well.  Hopefully, you've not trashed them in the process.

You can specify an alternative installation path for OpenSSL.  Details are in the INSTALL.md file.

Finally, problems installing openssh should be addressed to that project.
